### PR TITLE
Updates to TBI

### PR DESCRIPTION
**Pull request by dgrantkeane**

### CODE REVIEW

File: tbi.datacommons.io/portal/gitops.json [hunk 1]
```diff
@@ -117,11 +117,11 @@
       ]
     },
     "login": {
-      "title": "BRAIN Commons",
+      "title": "TBI Commons",
       "subTitle": "Explore, Analyze, and Share Data",
-      "text": "The BRAIN Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
+      "text": "The TBI Commons is a scalable cloud-based platform for computational discovery designed for the brain health community. The BRAIN Commons empowers the global research community by providing access to multi-modal data, state-of-the-art analytic tools and a secure interoperable system for data sharing to accelerate discovery and development of therapies, diagnostic tests, and other technologies for the treatment and prevention of diseases impacting the brain.",
```
Comment (mfshao) [inline tbi.datacommons.io/portal/gitops.json:122]: there are still some words saying `BRAIN Commons` in here and for some other buttons, do we want to change them all?

File: tbi.datacommons.io/portal/gitops.json [hunk 2]
```diff
@@ -117,11 +117,11 @@
...
       "contact": "If you have any questions about access or the registration process, please contact ",
-      "email": "support@braincommons.org"
+      "email": "support@datacommons.io"
     },
     "certs": {
```
Comment (mfshao) [inline tbi.datacommons.io/portal/gitops.json:126]: can remove this big block of certs if not needed

Comment (dgrantkeane) [reply tbi.datacommons.io/portal/gitops.json:126]: removed

Comment (dgrantkeane) [reply tbi.datacommons.io/portal/gitops.json:126]: removed

File: tbi.datacommons.io/portal/gitops.json (hunk 1 above)
Comment (dgrantkeane) [reply tbi.datacommons.io/portal/gitops.json]: i think i got them all now

File: tbi.datacommons.io/portal/gitops.json [hunk 3]
```diff
@@ -31,11 +31,11 @@
...
+    "appName": "The TBI Commons Portal",
     "index": {
       "introduction": {
-        "heading": "BRAIN Commons",
-        "text": "The BRAIN Commons supports the management, analysis and sharing of brain health data for the research community with the aim of accelerating discovery and development of diagnostics, treatment and prevention of diseases impacting the brain. The BRAIN Commons enables cross-project analyses through the harmonization of data to a common data model and data dictionary, and a cloud-based analysis workspace implementing Jupyter notebooks.",
+        "heading": "TBI Commons",
+        "text": "The TBI Commons supports the management, analysis and sharing of brain health data for the research community with the aim of accelerating discovery and development of diagnostics, treatment and prevention of diseases impacting the brain. The BRAIN Commons enables cross-project analyses through the harmonization of data to a common data model and data dictionary, and a cloud-based analysis workspace implementing Jupyter notebooks.",
         "link": "/submission"
       },
       "buttons": [
```
Comment (mfshao) [inline tbi.datacommons.io/portal/gitops.json:41]: there are still some in here

File: tbi.datacommons.io/portal/gitops.json [hunk 4]
```diff
@@ -160,7 +138,7 @@
       }
     ]
   },
-  "requiredCerts": ["privacy_policy"],
+  "requiredCerts": [],
   "featureFlags": {
     "explorer": true,
```
Comment (mfshao) [inline tbi.datacommons.io/portal/gitops.json:143]: maybe we should disable explorer for now since it won't work anyways (no guppy)

Comment (dgrantkeane) [reply tbi.datacommons.io/portal/gitops.json:143]: i think it's fine because no one should use it until data is in anyway

**Pull request merged**